### DISCUSSION TITLE
Is there anyway to have SSR in dev mode

### DISCUSSION THREAD

**Discussion by lejunyang** [vuejs/vitepress#4144] [Q&A] [answered]
I got some hydration issues with custom elements, and I found If I remove the comment \ around my custom element, it worked.
I have turned on DEBUG and prod hydration details, got warnings below, not really useful
<img width="1334" alt="image" src="https://github.com/user-attachments/assets/2c6c26a2-1c3e-4565-ae04-5d61b8b7651f">
I am trying to figure out whether it's coming from my custom elements or hydration, but it's really annoying that I have to run build and preview on every code change
Is there anyway to have SSR in dev mode(with code none compressed and hot building)

**Comment by brc-dd** [comment, marked as answer]
Currently no. It is planned but no ETA yet. Please track #4045.
